openmdao/core/component.py

```diff
@@ -165,6 +165,14 @@ def _add_variable(self, name, val, **kwargs):
         if isinstance(shape, int) and shape > 1:
             meta['shape'] = (shape,)
 
+        if 'low' in kwargs:
+            raise NameError("Used arg 'low' when adding variable '%s'. "
```

Comment:
Python raises `TypeError` for unexpected keywords, would we want to follow suit?